rocky/katalogus/views/boefje_setup.py

```diff
@@ -85,45 +67,154 @@ class AddBoefjeVariantView(BoefjeSetupView):
                 "text": "Boefje variant setup",
             },
         ]
 
         return context
+
+
+class EditBoefjeView(BoefjeSetupView):
+    """View where the user can update a Boefje."""
+
+    def setup(self, request, *args, **kwargs):
+        super().setup(request, *args, **kwargs)
+
+        plugin_id = self.kwargs.get("plugin_id")
+        katalogus = get_katalogus(self.organization.code)
+        self.plugin = katalogus.get_plugin(plugin_id)
+
+    def get_initial(self):
+        initial = super().get_initial()
+
+        consumes = []
+
+        for input_object in self.plugin.consumes:
+            consumes.append(input_object.__name__)
+
+        initial["name"] = self.plugin.name
+        initial["description"] = self.plugin.description
+        initial["oci_image"] = self.plugin.oci_image
+        initial["oci_arguments"] = " ".join(self.plugin.oci_arguments)
+        initial["boefje_schema"] = self.plugin.boefje_schema
+        initial["consumes"] = consumes
+        initial["produces"] = ", ".join(self.plugin.produces)
+        initial["scan_level"] = self.plugin.scan_level
+        initial["interval"] = self.plugin.interval
+
+        return initial
+
+    def form_valid(self, form):
+        """If the form is valid, redirect to the supplied URL."""
+        form_data = form.cleaned_data
+        plugin = create_boefje_with_form_data(form_data, self.kwargs.get("plugin_id"), self.plugin.created)
+
+        get_katalogus(self.organization.code).edit_plugin(plugin)
+
+        return redirect(
+            reverse("boefje_detail", kwargs={"organization_code": self.organization.code, "plugin_id": plugin.id})
+        )
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context["edit_boefje_name"] = self.plugin.name
+        context["return_to_plugin_id"] = self.plugin.id
+
+        context["breadcrumbs"] = [
+            {
+                "url": reverse("katalogus", kwargs={"organization_code": self.organization.code}),
+                "text": "KAT-alogus",
+            },
+            {
+                "url": reverse(
+                    "boefje_detail",
+                    kwargs={
+                        "organization_code": self.organization.code,
+                        "plugin_id": self.plugin.id,
+                    },
+                ),
+                "text": self.plugin.name,
+            },
+            {
+                "url": reverse(
+                    "edit_boefje",
+                    kwargs={
+                        "organization_code": self.organization.code,
+                        "plugin_id": self.plugin.id,
+                    },
+                ),
+                "text": 'Edit "' + self.plugin.name + '"',
+            },
+        ]
+
+        return context
+
+
+def create_boefje_with_form_data(form_data, plugin_id: str, created: str):
+    arguments = [] if form_data["oci_arguments"] == "" else form_data["oci_arguments"].split()
```

Comment:
A more Pythonic approach to check string emptiness is using the `not` operator (truthy value of an empty string is `False`)
```suggestion
    arguments = [] if not form_data["oci_arguments"] else form_data["oci_arguments"].split()
```